ssrando/ssrando
# Port ammo-availability feature from HDR

## What does this PR do?
Adds the `ammo-availability` setting from HDR. Should work exactly the same. Also moves the post-LMF bird statue into the same place that HDR puts it to make it more visible to player used to the base-game.

## How do you test this changes?
* With the setting set to "Plentiful", there should be an ammo pot added next to every bird statue, next to the gossip stone in Thrill Digger, and before the crawlspace in Lanayru Caves from the Gorge entrance.
* When set to "Useful", ammo pots will be added in useful places to avoid back-tracking. This is basically the same as things are now before this PR
* When set to "Vanilla", no extra ammo pots are added
* When set to "Scarce", all guaranteed ammo drops are removed. Only chance-based drops may give ammo and ofc Rupin's Gear Shop
